README.md

```diff
@@ -56,6 +56,19 @@ Contributions are welcome!
 3. We recommend to write a test which shows that the bug was fixed or that the feature works as expected.
 4. Send a pull request and bug the maintainer until it gets merged and published. :smiley:
 
+## Fix issue there're some applications that launched process exists immediately, so in close/quit function it throws exception Process Not Found.
```

Comment:
Same here, these should be in change log or similar places, README.md is not the right place.